references/depth/stereo/transforms.py

```diff
@@ -61,6 +61,22 @@ def forward(self, images, disparities, masks):
         return images, disparities, masks
 
 
+class ConverToGrayscale(torch.nn.Module):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def forward(
+        self,
+        images: T_STEREO_TENSOR,
+        disparities: Tuple[T_FLOW, T_FLOW],
+        masks: Tuple[T_MASK, T_MASK],
+    ) -> Tuple[T_STEREO_TENSOR, Tuple[T_FLOW, T_FLOW], Tuple[T_MASK, T_MASK]]:
+        img_left = F.to_grayscale(images[0], num_output_channels=3)
```

Comment:
Question, just wondering why we use `num_output_channels=3` ? Is it because we assume the model only accept 3-channel image?

Reply:
Yes, it's mainly for that reason and to not add additional boiler plate code in other sources. This way someone can easily finetune a model that was trained on RGB to Grayscale or Vice-Versa since the network deals with the exact same type of input.